app/views/RoomView/List.js

```diff
@@ -251,6 +252,9 @@ class List extends React.Component {
 					onEndReachedThreshold={5}
 					maxToRenderPerBatch={5}
 					windowSize={10}
+					scrollEventThrottle={20}
```

Comment:
I'm unsure about performance here.
`scrollEventThrottle={20}` means it's going to execute `toggleScrollToBottomButton` every 20ms.
We already have some performance issues on low-end devices.
This implementation is much better than https://github.com/RocketChat/Rocket.Chat.ReactNative/pull/735, but we still have to think about it.

Reply:
@diegolmello  I have changed the number to 8